operator/docker/Dockerfile.operator

```diff
@@ -15,10 +15,18 @@ FROM gcr.io/distroless/static-debian10@sha256:4433370ec2b3b97b338674b4de5ffaef8c
 # hadolint ignore=DL3006
 FROM ${BASE_DISTRIBUTION}
 
+ENV USER_UID=1001 \
```

Comment:
why not align with distroless (uid 65532) - as we won't easily be able to add UID/GID in a distroless future?